recipes/tabulate/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "tabulate" %}
+{% set version = " v1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/p-ranav/tabulate/archive/{{ version }}.tar.gz
+  sha256: 34eda3605e2ce0c7ca339fa2af6a0c77a5e49321b2f549ff5313f955273c8ac8
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
I'm just copying other people here. Should I skip build on windows? Why?

Reply:
you don't have to :) it just has to work, hehe